src/Store/InitializableStoreInterface.php

```diff
@@ -0,0 +1,13 @@
+<?php
+
+declare(strict_types=1);
+
+namespace PhpLlm\LlmChain\Store;
+
+interface InitializableStoreInterface extends StoreInterface
+{
+    /**
+     * @param array<mixed> $options
```

Comment:
```suggestion
     * @param array<string, mixed> $options
```

Reply:
I thought about this, but another store could be different, no?